addons/zotero/views.py

```diff
@@ -48,15 +48,16 @@ def _set_config(node_addon, user_addon, auth, **kwargs):
 
     def library_list(self):
         addon_short_name = self.addon_short_name
-        Provider = self.Provider
         @must_be_contributor_or_public
         @must_have_addon(addon_short_name, 'node')
         def _library_list(auth, node_addon, **kwargs):
             """ Returns a list of group libraries - for use with Zotero addon
             """
             limit = request.args.get('limit')
             start = request.args.get('start')
-            return Provider().library_list(node_addon, auth.user, limit, start)
+            return_count = request.args.get('return_count')
+            append_personal = request.args.get('append_personal')
+            return node_addon.get_folders(**{'limit': limit, 'start': start, 'return_count': return_count, 'append_personal': append_personal})
```

Comment:
Minor: No need to use `**{...}` here.

Reply:
@sloria, can you clarify what you want with this?  I don't want to use request.args.get('...') inside the get_folders method because this is a method on the model (and also v2 is using this).  The get_folders method is being used in both v1 and v2. v1 needs these limit/start/return_count/append_personal variables, v2 does not. Do you instead want all six variables to be passed in?

Reply:
I just meant that you can do 

```python
node_addon.get_folders(limit=limit, start=start, ...)
```

Reply:
ok, passing in the variables, got it.